atlasdb-impl-shared/src/main/java/com/palantir/atlasdb/sweep/SpecificTableSweeper.java

```diff
@@ -201,12 +198,14 @@ private static SweepResults getCumulativeSweepResults(TableToSweep tableToSweep,
         long minimumSweptTimestamp = Math.min(
                 tableToSweep.getPreviousMinimumSweptTimestamp().orElse(Long.MAX_VALUE),
                 currentIteration.getSweptTimestamp());
+        long timeInMillis = tableToSweep.getTimeInMillisPreviously() + currentIteration.getTimeInMillis();
```

Comment:
I don't think this is going to work right now - you also need to write out the cumulative time in millis to the `newProgress` value in `saveIntermediateSweepResults`.

Also need to check the `SweepProgress` can be read fine when this field isn't in the DB (when we first roll this out.

Reply:
We should probably use a new sweep progress table, to avoid breaking backwards compatibility if we go ahead with this.